docs/examples/buttongroup/defaultExample.js

```diff
@@ -0,0 +1,14 @@
+// @flow strict
+import { type Node } from 'react';
```

Comment:
Note that when the user views the code for this example, they'll see
```js
import {} from 'react';
```
You can avoid this by using a different type, which doesn't need to be imported:
```js
export default function Example(): React$Node {
```